src/aus.h

```diff
@@ -240,7 +243,21 @@ namespace librealsense
                     insert_device_to_device_manager(serial, name);
                 }
             }
-        } 
+        }
+
+        std::vector<std::uint8_t>  get_data()
+        {
+            json j;
+            j["librealsense_version"] = _librealsense_version;
+            j["os_name"] = _os_name;
+            j["platform_name"] = _platform_name;
+
+            for ( const auto & pair : _mp )
+            {
+                j[pair.first] = std::to_string(pair.second->get());
+            }
+            return json::to_ubjson(j);
```

Comment:
What does `to_ubjson` means?

Reply:
It serializes a given JSON value to a byte vector using the UBJSON (Universal Binary JSON) serialization format.